iroh/src/magicsock/relay_actor.rs

```diff
@@ -173,6 +157,41 @@ impl ConnectedRelayActor {
         Ok(())
     }
 
+    async fn handle_actor_msg(&mut self, msg: ConnectedRelayMessage) -> bool {
+        trace!("tick: inbox: {:?}", msg);
+        match msg {
```

Comment:
are you sure all these should be preferred over regular writes? it seems some of these will trigger additional sending, eg `Ping` and `NotePreferred`, or amy I missing sth?

Reply:
They are not being preferred, the `tokio::select!` is not biased.  The ones that do trigger a send are indeed blocking.  I don't like it, but it is not a regression so in the interest of taking small steps I was only improving the SendPacket path.